### PR TITLE
Feat/cli global flags

### PR DESCRIPTION
**Pull request by MildTomato**
Added global flags section to CLI

Screenshot 2022-12-14 at 4 17 11 PM

### CODE REVIEW

File: apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+// @ts-expect-error
+import spec from '~/../../spec/cli_v1_commands.yaml' assert { type: 'yaml' }
+import Param from '~/components/Params'
+import Options from '~/components/Options'
+
+const CliGlobalFlagsHandler = () => {
+  return (
+    <RefSubLayout.EducationRow className="not-prose">
+      <>
```
Comment (sweatybridge) [inline apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx:11]: unnecessary pseudo element?

File: apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <RefSubLayout.Details>
+          <h3 className="text-lg text-scale-1200 mb-3">Flags</h3>
+          <ul className="">
+            {spec.flags.map((flag) => {
+              console.log('flag', flag)
```
Comment (sweatybridge) [inline apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx:16]: Would this spam browser console? may be use `console.debug`

**Review by sweatybridge — APPROVED** [review]

File: apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx (hunk 2 above)
Comment (MildTomato) [reply apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx:16]: oops

Comment (MildTomato) [reply apps/docs/components/reference/enrichments/cli/CliGlobalFlagsHandler.tsx]: fixed

**Pull request merged**